Formula/tektoncd-cli.rb

```diff
@@ -0,0 +1,24 @@
+class TektoncdCli < Formula
+  desc "CLI for interacting with TektonCD"
+  homepage "https://github.com/tektoncd/cli"
+  url "https://github.com/tektoncd/cli/archive/v0.2.2.tar.gz"
```

Comment:
Can you use tag-revision format? I am trying to push all golang formula in that way. :)
```suggestion
  url "https://github.com/tektoncd/cli.git",
    :tag      => "v0.2.2",
    :revision => "f18e2d0bdb7d05923d476bdf62bd706f0d3bb9e0"
```

Reply:
Actually tarbals are the preferred source for homebrew formula. Unless there is a good reason to use git (submodules for example) you should use those.